opentelemetry-api/opentelemetry/backend.py

```diff
@@ -0,0 +1,134 @@
+# Copyright 2019, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+"""
+The OpenTelemetry backend module provides access to a selected implementation (backend) of the
+OpenTelemetry API.
+
+By default, if you call a getter function (e.g., :func:`tracer`) and the corresponding setter (e.g.,
+:func:`set_tracer`) wasn't called, you will get a default implementation, which is selected as
+follows:
+
+    1. If the environment variable :samp:`OPENTELEMETRY_PYTHON_BACKEND_{getter-name}` (e.g.,
+       ``OPENTELEMETRY_PYTHON_BACKEND_TRACER``) is set to an nonempty value, an attempt is made to
+       import a module with that name and call a function ``get_opentelemetry_backend_impl`` in it.
+       The function receives the API type that is expected as an argument and should return an
+       instance of it (e.g., the argument is :class:`opentelemetry.trace.Tracer` and the function
+       should return an instance of a :class:`~opentelemetry.trace.Tracer` (probably of a derived
+       type).
+    2. If that variable is not set, ``OPENTELEMETRY_PYTHON_BACKEND_DEFAULT`` is tried instead.
```

Comment:
And what does this value stands for? The same as the previous one? Boolean?

Reply:
Yes, the same as the previous one. If some PaaS platform just want's to override tracing, it would just set `OPENTELEMETRY_PYTHON_BACKEND_TRACER` (note: that is called `OPENTELEMETRY_PYTHON_IMPLEMENTATION_TRACER` now). If it wants to override everything, it would set the `DEFAULT` variant (unsupported objects will be filled in with the no-op implementation by the loader module)